src/main/java/thestonedturtle/partypanel/ui/PlayerPanel.java

```diff
@@ -68,6 +77,8 @@ public class PlayerPanel extends JPanel
 	private final PlayerSkillsPanel skillsPanel;
 	private final PlayerPrayerPanel prayersPanel;
 
+	private boolean showInfo = true;
```

Comment:
This should probably be off by default to ensure you can see most (if not all) party members when first viewing the panel.

This could also be changed to a config option if you think this is something that the end-user may want to control.

Reply:
I agree it should be off by default to show the most players. As for a user config, I agree it would be nice. I tried a simple solution for it and I didn't get it working properly, so for now it will be something I consider later.

Reply:
Add `@Getter` to the config in the main plugin class, and add `@Setter` to this value. When creating a new `PlayerPanel` pass `plugin.getConfig().XXX` as another constructor argument.

Add a new method to the `PartyPanel` that will take a boolean value and update the `showInfo` for all values in the `playerPanelMap`.

In the `onConfigChanged` inside the main plugin add an if for the new config `XXX` and call the PartyPanel method.

Reply:
Thanks for the push getting it to fully work. Putting a function there to set all playerPanelMap values was what I needed to add. I also had to make `PlayerPanel.updatePanel()` a public function because it needed to be called from there after changing the values.